.github/workflows/darwin.yaml

```diff
@@ -51,30 +54,33 @@ jobs:
             - name: Checkout submodules & Bootstrap
               uses: ./.github/actions/checkout-submodules-and-bootstrap
               with:
-                platform: darwin
-                bootstrap-log-name: bootstrap-logs-framework-${{ matrix.options.flavor }}
+                  platform: darwin
+                  bootstrap-log-name:
+                      bootstrap-logs-framework-${{ matrix.options.flavor }}
             - name: Block zap-cli from being used
               env:
-                PW_ENVSETUP_NO_BANNER: 1
+                  PW_ENVSETUP_NO_BANNER: 1
               run: |
-                # Framework builds are NOT expected to require zap-cli
-                scripts/run_in_build_env.sh 'rm -- "$(which zap-cli)"'
-                # run_in_build_env.sh is used to ensure PATH is set to something that would otherwise find zap-cli
-                scripts/run_in_build_env.sh '(zap-cli --version && exit 1) || exit 0'
+                  # Framework builds are NOT expected to require zap-cli
+                  scripts/run_in_build_env.sh 'rm -- "$(which zap-cli)"'
+                  # run_in_build_env.sh is used to ensure PATH is set to something that would otherwise find zap-cli
+                  scripts/run_in_build_env.sh '(zap-cli --version && exit 1) || exit 0'
             - name: Build
               working-directory: src/darwin/Framework
               run: xcodebuild -target "Matter" ${{ matrix.options.arguments }}
 
     tests:
         name: Run framework tests
         if: github.actor != 'restyled-io[bot]'
-        needs: [ framework ] # serialize to avoid running to many parallel macos runners
+        needs: [framework] # serialize to avoid running to many parallel macos runners
         runs-on: macos-13
         strategy:
             matrix:
                 options: # We don't need a full matrix
                     - flavor: asan
-                      arguments: -enableAddressSanitizer YES -enableUndefinedBehaviorSanitizer YES
+                      arguments:
+                          -enableAddressSanitizer YES
+                          -enableUndefinedBehaviorSanitizer YES
```

Comment:
This reformat is odd.  It's making it look like a YAML array, but it's just a string, no?